pkg/systemd/logs.js

```diff
@@ -93,6 +93,8 @@ $(function() {
         }
     }
 
+    var services_list = [];
```

Comment:
Not sure that journal messages are the best source to build this list and it's not like we load the entire journal here. We should probably either load the list directly from dbus, and/or allow users to type free form text in the dropdown.

Reply:
hum... good point. 
You are right, the list is not completely loaded at start but in order to do load it I have to query it separately and the dropdown list will be giant.
Should I switch the field to a semicolon separated free form ?

Reply:
@andreasn do you have a suggestion here? Is a big dropdown list ok or is there a different way we should handle it?

Reply:
Jost for the record, this is a preview of the dropdown list when there are a lot of services

![image](https://user-images.githubusercontent.com/3775353/34520013-fc01961a-f06d-11e7-928c-c2e47dd3cec4.png)

Reply:
@andreasn or @garrett, any thoughts?

Reply:
Big dropdown is fine to me for now. We can do an inline search as a followup.

Reply:
I'd just like to reiterate that we should be using `journalctl --field=SYSLOG_IDENTIFIER` here to build this menu.